app/src/main/cpp/switch/kernel/svc.h

```diff
@@ -32,40 +28,99 @@ namespace lightSwitch::constant {
         constexpr uint8_t PersonalMmHeapSize = 0x10;
         constexpr uint8_t PersonalMmHeapUsage = 0x11;
         constexpr uint8_t TitleId = 0x12;
+        // 4.0.0+
+        constexpr uint8_t PrivilegedProcessId = 0x13;
         // 5.0.0+
         constexpr uint8_t UserExceptionContextAddr = 0x14;
         // 6.0.0+
         constexpr uint8_t TotalMemoryAvailableWithoutMmHeap = 0x15;
         constexpr uint8_t TotalMemoryUsedWithoutMmHeap = 0x16;
     };
+    namespace status {
+        constexpr uint32_t success = 0x0; //!< "Success"
+        constexpr uint32_t unimpl = 0x177202; //!< "Unimplemented behaviour"
+    }
```

Comment:
These error codes are also used in services, so you should probably consider moving them